shell/src/main/java/alluxio/cli/fsadmin/report/CapacityCommand.java

```diff
@@ -58,7 +59,7 @@
   public CapacityCommand(BlockMasterClient blockMasterClient, PrintStream printStream) {
     mBlockMasterClient = blockMasterClient;
     mPrintStream = printStream;
-    mStringBuilder = new StringBuilder();
+    initValues();
```

Comment:
why having a separate method?
Typically the best practice is that Java constructors should not call instance method

Reply:
I place the initVariables method inside collectWorkerInfo method instead because it inits the member variables that we save worker info to. That way may be better.

The initVariables method has nothing to do with the constructor input, so maybe it will be better not put here.

Thanks for the information, I will not put the non-static method in constructors in the future.